Quaver.Shared/Screens/Edit/UI/Playfield/EditorHitObject.cs

```diff
@@ -145,14 +145,16 @@ public void SetPosition()
         /// </summary>
         public Texture2D GetHitObjectTexture()
         {
-            var index = SkinMode.ColorObjectsBySnapDistance ? HitObjectManager.GetBeatSnap(Info, Info.GetTimingPoint(Map.TimingPoints)) : 0;
+            var snap = SkinMode.ColorObjectsBySnapDistance ? HitObjectManager.GetBeatSnap(Info, Info.GetTimingPoint(Map.TimingPoints)) : 0;
+            var index = SkinMode.NoteHoldHitObjects[Info.Lane - 1].Count / 9 * snap;
```

Comment:
Not really a fan of the `/ 9 * snap` everywhere. 

1. It uses a magic number.
2. It's not always 9. The current behavior in the game allows the user to have anywhere from 1-9 snaps set in their skin.

Code needs to be cleaner and abstracted, so we're not doing this everywhere.

Reply:
Fixed, it now doesn't rely on a magic number. It also now work for whatever snap set in the spritesheet. (also updated test skin to include diverse spritesheet)